paper-card.html

```diff
@@ -237,6 +237,10 @@
         return image ? 'false' : 'true';
       },
 
+      created: function() {
+        this.classList.add('paper-material');
```

Comment:
(Poor @ keanu, getting all them notifications...)

This is also present in https://github.com/PolymerElements/paper-button/pull/134 (along with an issue https://github.com/PolymerElements/paper-button/issues/144), so we should make sure that this is consistent across all paper-elements.

I don't know if `attached` is the most appropriate (since it is run on every attachment). Maybe `ready` is better? (Though I don't know if that actually makes a huge difference in practice.)